### PR TITLE
Fixes #3288

### LINKED ISSUE #3288: Images missing or not displaying on French pages
## La page *Flexbox - Apprendre le développement web*

La version française de la page sur Flexbox n'affiche pas les images (on ne voit que le texte alternatif). Cette erreur se retrouve également dans toutes les versions non-anglaises avec l'image **flex_terms.png** (celle-ci a été transférée à #3294).

- Aucune image n'est affichée ici : https://developer.mozilla.org/fr/docs/Learn/CSS/CSS_layout/Flexbox

Par ailleurs, quelques images ont été ajoutées à la page sur Flexbox en anglais (https://developer.mozilla.org/en-US/docs/Learn/CSS/CSS_layout/Flexbox) depuis la dernière révision de la version française, dont les suivantes si je ne me trompe pas :
- bih741v.png
- flexbox_center_space-around.png
- flexbox_first-child_flex-end.png
- flexbox-example7.png (je crois que ce fichier devrait être nommé flexbox-example**6**.png, au moins en anglais)

Il y a aussi deux images avec le même attribut `src`, **Exemple_flexbox_1.png**

### PR DESCRIPTION
**Pull request by SphinxKnight**
Thanks @cw118 for the report!

### CODE REVIEW

**Comment by cw118** [comment]
Thanks so much for fixing this!

I just noticed some repetitive and possibly unneeded markdown in **css\_layout/flexbox/index.md** at lines 78 and 79 _(after edits from this PR, or lines 90 and 91 without)_:
- There are back-to-back right and left-flank asterisks for bold text on several occasions (e.g. `L'**axe principal** **(main axis)** est l'axe...`).
- There are also some random escaped asterisks (`\*\*`), and a few extra ones where the text probably doesn't need to/shouldn't be bold, though I'm not entirely sure what the intent was.

Wasn't sure whether I should comment here or directly on the files, but I hope this helps :) Thank you again for this fix!

**Comment by SphinxKnight** [comment]
> Wasn't sure whether I should comment here or directly on the files, but I hope this helps :) Thank you again for this fix!
Good catch, I'll do a "linting"/typofixing review of those pages in the coming hours. Let's fix this while this PR is open.

File: files/fr/learn/css/css_layout/flexbox/index.md [hunk 1]
```diff
@@ -75,21 +63,21 @@ section {
...
-- L'**axe croisé** \*\* **est l'axe perpendiculaire à l'axe principal, càd à la direction dans laquelle sont disposés les éléments flex. Le début et la fin de cet axe sont appelés le début **(cross start)** et la fin (**cross end\*\*) de l'axe croisé.
-- L'élément parent dont la propriété est `display: flex`  ({{htmlelement("section")}} dans notre exemple) est appelé le **conteneur flex** (**flex container**).
+- L'**axe croisé** est l'axe perpendiculaire à l'axe principal, c'est-à-dire à la direction dans laquelle sont disposés les éléments flex. Le début et la fin de cet axe sont appelés le début (**cross start**) et la fin (**cross end**) de l'axe croisé.
+- L'élément parent dont la propriété est `display: flex` ({{htmlelement("section")}} dans notre exemple) est appelé le **conteneur flex** (**flex container**).
 - Les éléments disposés en tant que boîtes flexibles à l'intérieur du conteneur flex sont appelés **éléments flex (flex items)** (les éléments {{htmlelement("article")}}} dans notre exemple).
```
Comment (cw118) [inline files/fr/learn/css/css_layout/flexbox/index.md:81]: There seems to be an extra closing curly bracket `}` for the HTML article macro (it's currently written as `{{htmlelement("article")}}}`, making the page display *"les éléments `<article>`**}**"*), line 81

File: files/fr/learn/css/css_layout/flexbox/index.md [hunk 2]
```diff
@@ -153,7 +141,7 @@ article {
 }
 ```
 
-Il s'agit d'une valeur de proportion, sans unité, définissant la quantité d'espace disponible que chaque élément flex prendra le long de l'axe principal. Dans ce cas, nous donnons à chaque élément {{htmlelement("article")}}} une valeur de 1, ce qui signifie qu'ils prendront tous une portion égale de l'espace libre après le calcul du remplissage et de la marge. Cette valeur représente une proportion, càd que le fait de donner une valeur de 400 000 simultanément à tous les éléments flex aurait exactement le même effet.
+Il s'agit d'une valeur de proportion, sans unité, définissant la quantité d'espace disponible que chaque élément flex prendra le long de l'axe principal. Dans ce cas, nous donnons à chaque élément {{htmlelement("article")}}} une valeur de 1, ce qui signifie qu'ils prendront tous une portion égale de l'espace libre après le calcul du remplissage et de la marge. Cette valeur représente une proportion, c'est-à-dire que le fait de donner une valeur de 400 000 simultanément à tous les éléments flex aurait exactement le même effet.
```
Comment (cw118) [inline files/fr/learn/css/css_layout/flexbox/index.md:144]: Another extra closing curly bracket } for the HTML article macro here (line 144)

File: files/fr/learn/css/css_layout/flexbox/index.md [hunk 3]
```diff
@@ -320,7 +312,7 @@ article:nth-of-type(3) div:first-child {
 }
 ```
 
-Enfin, nous définissons un dimensionnement des boutons, et plus précisément nous leur donnons une valeur flex de 1. L'effet obtenu est très intéressant ; vous l'observerez en modifiant la largeur de la fenêtre du navigateur. Les boutons prennent autant d'espace qu'il leur est permis, et sont si possible disposés sur la même ligne ; mais si ce n'est pas possible, il "descendent" pour créer de nouvelles lignes.
+Enfin, nous définissons un dimensionnement des boutons, et plus précisément nous leur donnons une valeur flex de 1. L'effet obtenu est très intéressant ; vous l'observerez en modifiant la largeur de la fenêtre du navigateur. Les boutons prennent autant d'espace qu'il leur est permis, et sont si possible disposés sur la même ligne ; mais si ce n'est pas possible, ils "descendent" pour créer de nouvelles lignes.
 
 ```css
 button {
```
[the remark this answers is not in the document]
Comment (cw118) [reply files/fr/learn/css/css_layout/flexbox/index.md:318]: The *Testez vos compétences* section is also missing (meant to be in between *Compatibilité des navigateurs* and *Résumé*), but I saw that the [Testez vos compétences : Flexbox](https://developer.mozilla.org/fr/docs/Learn/CSS/CSS_layout/Flexbox_skills) page is still incomplete and not fully translated, so perhaps such a section should be added here after the *Testez vos compétences : Flexbox* page has been updated?

**Comment by cw118** [comment]
Apologies for yet another comment and all the notifications I must have triggered... If there's a better way to organize everything (e.g. creating more issues or pull requests) please do let me know—I really hope I'm not troubling you! 

Some mistakes I've noticed in *Grilles* so far (*css_layout/grids*):

- The links in the table (**lines 11-32**) were translated to French; as a result, the links to *Introduction aux CSS* and *Styles de boîtes* fallback on the en-US pages
- The same block of CSS code appears twice, at **lines 62 and 99** (to match the en-US page, the code block at line 62 would be kept and the other removed):
``` css
.container {
  display: grid;
  grid-template-columns: 200px 200px 200px;
}
```
- **Line 172** of *Espaces entre pistes* lists obsolete CSS properties, one of which is currently used in the code example as well (***Edit*:** these obsolete properties are found in several code samples on the *Grilles* page, and the French `row-gap` page has a broken EmbedLiveSample):
  - `grid-column-gap` is now `column-gap`
  - `grid-row-gap` is now `row-gap`
  - `grid-gap` is now `gap` (also seen in the code example on line 178 and various others)
- The HTML at **line 207** should be hidden
- **Line 221** has an escaped asterisk (_\*gap_), not sure if this'll be fixed by the "starry-starry-sky" PR :)
- **Line 248** has a few extra spaces (in between bolded words) and a typo:
  - *Nous n'avons jusqu'à présent défini que **des  chaînes**...*
  - *La* cahîne *explicite et la **trame  implicite**...* (*chaîne* and extra space)

**Comment by SphinxKnight** [comment]
> Apologies for yet another comment and all the notifications I must have triggered... If there's a better way to organize everything (e.g. creating more issues or pull requests) please do let me know—I really hope I'm not troubling you!

No worries at all.
I think it would be best to keep this PR's scope as intended (i.e. fixing the images).

I'll open an issue for each page that you mentioned where "update work" is needed including your comments so that we can address each one :)

I hope that's OK with you, let me know if not.
If it is, I'll merge this one so that there are fewer conflict chances for the upcoming PRs.

**Comment by cw118** [comment]
> I hope that's OK with you, let me know if not. If it is, I'll merge this one so that there are fewer conflict chances for the upcoming PRs.

Definitely! Thank you so much for your patience and hard work—I'd be happy to help with these pages as well :)

**Pull request merged**